inst/htmljs/animint.js

```diff
@@ -1609,29 +1610,50 @@ var animint = function (to_select, json_file) {
       moreActions(e)
     };
     doActions(enter);  // DO NOT DELETE!
+
+    //adding tooltip using tippy.js
+
     var has_tooltip = g_info.aes.hasOwnProperty("tooltip");
-    if(has_clickSelects || has_tooltip || has_clickSelects_variable){
-      var text_fun;
-      if(has_tooltip){
-        text_fun = function(d){
-	  return d.tooltip;
-	};
-      }else if(has_clickSelects){
-	text_fun = function(d){
+    if (has_clickSelects || has_tooltip || has_clickSelects_variable) {
+      var tooltipText;
+
+      // defining the tooltip text based on the aesthetic
+      if (has_tooltip) {
+        // Using the custom tooltip text from aes(tooltip)
+        tooltipText = function (d) {
+          return d.tooltip;
+        };
+      } else if (has_clickSelects) {
+        // Using the default tooltip text for clickSelects
+        tooltipText = function (d) {
           var v_name = g_info.aes.clickSelects;
           return v_name + " " + d.clickSelects;
-	};
-      }else{ //clickSelects_variable
-	text_fun = function(d){
-	  return d["clickSelects.variable"] + " " + d["clickSelects.value"];
-	};
-      }
-      // if elements have an existing title, remove it.
+        };
+      } else { // clickSelects_variable
+        // Using the default tooltip text for clickSelects.variable
+        tooltipText = function (d) {
+          return d["clickSelects.variable"] + " " + d["clickSelects.value"];
+        };
+      }
```

Comment:
here it looks like you just changed text_fun to tooltipText, but there are lots of other lines changed too.
please avoiding changing lines that are not necessary.
can you please undo all of these changes, and instead use text_fun as in the original code?

Reply:
click Files Changed tab in github to see what changes will appear for me to review